apis/r/R/SOMADataFrame.R

```diff
@@ -472,26 +472,36 @@ SOMADataFrame <- R6::R6Class(
     #' @param new_domain A named list, keyed by index-column name, with values
     #' being two-element vectors containing the desired lower and upper bounds
     #' for the domain.
-    #' @return No return value
-    tiledbsoma_upgrade_domain = function(new_domain) {
-      # stopifnot("'new_domain' must be CODE ME UP PLZ" = ...
+    #' @param check_only If true, does not apply the operation, but only reports
+    #' whether it would have succeeded.
+    #' @return No return value if `check_only` is `FALSE`. If `check_only` is `TRUE`,
+    #' returns the empty string if no error is detected, else a description of the error.
```

Comment:
This isn't entirely correct since it's currently invisibly returning `reason_string` if `check_only` is not `TRUE`; we should either update the docs or invisibly return `NULL` if `check_only` is not `TRUE`